webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -1638,7 +1638,7 @@ object RudderConfig extends Loggable {
     , RUDDER_BATCH_DYNGROUP_UPDATEINTERVAL
   )
 
-  private[this] lazy val dynGroupUpdaterService = new DynGroupUpdaterServiceImpl(roLdapNodeGroupRepository, woLdapNodeGroupRepository, queryProcessor)
+  private[this] lazy val dynGroupUpdaterService: DynGroupUpdaterServiceImpl = new DynGroupUpdaterServiceImpl(roLdapNodeGroupRepository, woLdapNodeGroupRepository, queryProcessor)
```

Comment:
why the typing ?

Reply:
almost all other lazy values are typed, it works without but since i added it

Reply:
i can remove it if you want

Reply:
it's not necessary, it was a genuine question

Reply:
I removed it so it does not make another change (typing is not done for other lines ...)